src/site/xdoc/server/config-users.xml

```diff
@@ -33,6 +33,17 @@
 
       <p>Consult <a href="https://github.com/apache/james-project/tree/master/server/app/src/main/resources/usersrepository.xml">usersrepository.xml</a> in GIT to get some examples and hints.</p>
 
+      <p>
+        An user has two attributes: username and password. A valid user should satisfy these criteria:
+      </p>
+      <ul>
+        <li>username and password cannot be null or empty</li>
+        <li>username should not longer than 255 characters</li>
+        <li>username can not contain '/'</li>
+        <li>username can not contain multiple domain delimiter('@')</li>
+        <li>An username can have only local part. E.g.'myUser'</li>
+        <li>An username can have domain part, and the domain part should be concatenated after a domain delimiter('@'). E.g. 'myuser@james.org'</li>
```

Comment:
A username

Reply:
hmmm no? An username is correct I think?

Reply:
https://www.englishforums.com/English/AUsernameOrAnUsername/pxgzm/post.htm

```
"A username" is the correct version. I believe this is because the u in user actually sounds like a consonant - "ju", so it is like "juser". The point is not what letter it starts with, but what sound. "Username" starts with a consonant sound, "yoo", so "a username" is correct.
```

Reply:
now I know my grammar is wrong so badly

Reply:
No big deal, that is why we have reviews ;-)

Reply:
If English starts having exceptions like in French... :D Fair point, I didn't know, thanks for that^^